src/components/sidebar.js

```diff
@@ -8,34 +8,20 @@ import { config } from "./config";
 import { configGuides } from "./config-guides";
 import Icon from "./icon";
 
-class Sidebar extends React.Component {
-  renderTransformedToc(siblings, targetLocation) {
-    const md = MarkdownIt();
-
-    return (
-      <ul className="Sidebar-toc">
-        {
-          siblings.map((sibling, id) => {
-            if (Array.isArray(sibling)) {
-              return (
-                <li className="Sidebar-toc-item" key={id}>
-                  {this.renderTransformedToc(sibling, targetLocation)}
-                </li>
-              );
-            }
+class SidebarListItem extends React.Component {
```

Comment:
I don't know what our style preferences are on separating out 1 off components... Didn't really see the need to create a separate file but I can!